docs/output-kafka.asciidoc

```diff
@@ -275,6 +276,14 @@ to allow other records to be sent so that the sends can be batched together.
 
 The maximum size of a request
 
+[id="plugins-{type}s-{plugin}-message_header"]
+===== `message_key`
+
+  * Value type is <<string,string>>
+  * There is no default value for this setting.
+
+The key for the message.
+
```

Comment:
This will need to be updated to reflect the actual details of the new field. It's defined as a hash (key/value map), so likely something like:

```suggestion
===== `message_header`

  * Value type is <<hash,hash>>
  ** Keys are header names, and must be <<string,string>>
  ** Values are header values, and must be <<string,string>>
  ** Both Keys and Values support interpolation from event field values
  * There is no default value for this setting.

The headers to send with the message, such as:

[source]
----
    message_header => { "event_timestamp" => "%{@timestamp}" }
----
```

Reply:
Hi @yaauie , the documentation was updated according to your proposal, the key is now a static string. and a type check for the keys in message_headers had been added to the register function.